src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -140,8 +140,11 @@ internal override void OnModelUpdated(Model modelOpt)
             else
             {
                 var selectedItem = modelOpt.SelectedItemOpt;
-                var viewSpan = selectedItem == null ? (ViewTextSpan?)null : modelOpt.GetViewBufferSpan(selectedItem.Span);
-                var triggerSpan = viewSpan == null 
+                var viewSpan = selectedItem == null || selectedItem.Document == null
```

Comment:
Can you add a comment to explain why the `selectedItem.Document` being `null` means that we can't use its `.Span` and what setting the `triggerSpan` to `null` will end up doing?